lib/registries/addon/components/editable-field/template.hbs

```diff
@@ -1,16 +1,79 @@
-<div local-class='Editable'>
-    <h4>{{@title}}</h4>
-    {{#if @registration.userHasAdminPermission}}
-        <OsfButton
-           data-test-edit-button
-           aria-label={{t 'general.edit'}}
-           data-analytics-name='Edit'
-           local-class='EditButton'
-           @type='link'
-           @onClick={{action (mut this.shouldShowEditable) true}}
-        >
-            <FaIcon @icon='pencil' @fixedWidth={{true}} />
-        </OsfButton>
+<this.manager as |manager|>
+    <div local-class='Editable'>
+        {{#if manager.shouldShowTitle}}
+            <h4>{{@title}}</h4>
+        {{/if}}
+        {{#if @registration.userHasAdminPermission}}
+            <OsfButton
+               data-test-edit-button={{@name}}
+               aria-label={{t 'general.edit'}}
+               data-analytics-name='Edit'
+               local-class='EditButton'
+               @type='link'
+               @onClick={{action (mut this.shouldShowEditable) true}}
+            >
+                <FaIcon @icon='pencil' @fixedWidth={{true}} />
+            </OsfButton>
+        {{/if}}
+    </div>
+
+    {{#if manager.shouldShowEmptyFieldText}}
+        {{manager.emptyFieldText}}
     {{/if}}
-</div>
-{{yield (hash shouldShowEditable=this.shouldShowEditable hideEditable=(action this.hideEditable))}}
+
+    {{#unless manager.fieldIsEmpty}}
+        {{yield (hash
+            display=(component 'editable-field/block')
+            manager=manager
+        )}}
+    {{/unless}}
+
+    <BsModal
+        @open={{this.shouldShowEditable}}
+        @onHide={{action manager.cancel (action this.hideEditable)}}
+        as |modal|
+    >
+        <modal.header>
+            <h3 class='modal-title'>
+                {{t 'registries.registration_metadata.edit_field' field=@name}}
+            </h3>
+        </modal.header>
+        <modal.body>
+            <div data-test-editable-view-modal>
+                {{yield (hash
+                    edit=(component 'editable-field/block')
+                    manager=manager
+                    shouldShowEditable=this.shouldShowEditable
+                    hideEditable=(action this.hideEditable)
+                )}}
+                {{#if manager.fieldChanged}}
```

Comment:
We shouldn't hide form buttons like this -- make them visible from the start.

They should always be able to click "cancel" to close the modal, and clicking "save" when there are no changes shouldn't hurt anything.